go/vt/dbconnpool/connection_pool.go

```diff
@@ -118,6 +121,7 @@ func (cp *ConnectionPool) connect(ctx context.Context) (pools.Resource, error) {
 	}
 	return &PooledDBConnection{
 		DBConnection: c,
+		timeCreated:  time.Now(),
```

Comment:
I believe this is not required for PooledDBConnection. They do not need to expire